packages/network/README.md

```diff
@@ -0,0 +1,158 @@
+# Network API
+<!--
+  content should remain synchronized with
+  https://github.com/Agoric/documentation/blob/HEAD/main/repl/networking.md
```

Comment:
404 Not Found, did you intend a different URL?
```suggestion
  https://github.com/Agoric/documentation/blob/main/main/reference/repl/networking.md
```

Reply:
It was already broken when I got here, but thanks for noticing; updated!